Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS build
```

Comment:
The use of 'AS build' is correct for multi-stage builds, but ensure that all necessary build dependencies are installed in this stage if required.